src/bindings/python/test.py

```diff
@@ -50,7 +51,43 @@
 
                 print("  %s %12s%12f" % (prefix, candidate['plate'], candidate['confidence']))
 
+finally:
+    if alpr:
+        alpr.unload()
+
+
+try:
+    alpr = Alpr(options.country, options.config, options.runtime_data)
```

Comment:
Does it make sense to create a cv argument?  If you write a python script i'll assume you will make a default choice of either using the byte array or the numpy array. Not sure what would be the point of having both as an option.